src/app/route/ProductPage/ProductPage.container.js

```diff
@@ -171,14 +188,15 @@ export class ProductPageContainer extends PureComponent {
             this.getRequiredProductOptions(items);
         }
 
-        if (id !== prevId) {
+        if (id !== prevId || (id === prevId && productName !== metaTitle)) {
```

Comment:
This check `productName !== metaTitle` makes no sense, I believe they might be different.

Reply:
It's required to set the browser tab title with product name.